README.md

```diff
@@ -1,122 +1,91 @@
-We hope that eventually there will be Pangeo Forge bakeries running in most regions of major cloud providers.
-
-![diagram](pangeo-forge-diagram.png)
+We envision a vibrant, dynamic library of open-access ARCO data stored in public clouds, shared among thousands of scientists and directly accessible to data-proximate computing.
 
+However, manually populating such a library would be prohibitively difficult and tedious. Instead, we are building Pangeo Forge to automate the production of ARCO data and enable the crowdsourcing of such a data library.
 
-## Subprojects
+Execution environments, managed by different institutions, are used to automate and scale the pipeline of downloading original files from their source (e.g. FTP, HTTP, or OpenDAP), combining them into one coherent dataset (e.g. using xarray), and writes the data in a cloud optimized format (e.g. Zarr) to cloud storage in a streaming fashion. Object storage may also be considered a data store but the download step may be skipped in cases where the execution environment is deployed in the same region as the same cloud environment executing the recipe.
 
-Pangeo Forge brings together a number of smaller subprojects to implement this vision.
-The currently-active subprojects are
+## Technical Concepts and Architecture
 
-### pangeo-forge
+Pangeo Forge’s high-level infrastructure includes:
 
-<https://github.com/pangeo-forge/pangeo-forge>
+1. Recipes
+2. Execution environments
 
-![CI](https://github.com/pangeo-forge/pangeo-forge/workflows/CI/badge.svg)
-![Codecov](https://img.shields.io/codecov/c/github/pangeo-forge/pangeo-forge)
-[![Documentation Status](https://readthedocs.org/projects/pangeo-forge/badge/?version=latest)](https://pangeo-forge.readthedocs.io/en/latest/?badge=latest)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+### Recipes
 
-The `pangeo_forge` python package provides the core API for creating Recipes.
-All of the "business logic" for how to extract, transform, and load data lives in this library; as such, it is the focal point of Pangeo Forge development.
+In Pangeo Forge, recipes generate analysis-ready cloud-based copies of datasets in a cloud-optimized format like Zarr. A recipe defines how to transform data in one format + location into another format + location. Recipes reduce the technical burden on scientists, enabling them to contribute without needing in-depth knowledge of cloud infrastructure. 
 
-### staged-recipes
+The standalone python package [`pangeo-forge-recipes`](https://github.com/pangeo-forge/pangeo-forge-recipes) consists of configurable and chainable algorithms for ARCO data production. These recipes can be used in a standalone fashion, without integration with an execution environment or they can be deployed into an execution environment for offline and scaling features.
 
-<https://github.com/pangeo-forge/staged-recipes>
+### Execution Environments (aka Bakeries)
```

Comment:
Should we still call them bakeries? Seems like we've moved on to execution environments.

Reply:
I prefer `Execution Environments` as well